build.gradle

```diff
@@ -240,6 +240,12 @@ task deploy() {}
 
 subprojects {
 
+  build {
```

Comment:
Is this this needed? Looks like it was just for testing

Reply:
agreed